hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/issues/Issue2319Spec.java

```diff
@@ -44,28 +44,28 @@
 public class Issue2319Spec extends HapiSuite {
     private static final Logger log = LogManager.getLogger(Issue2319Spec.class);
 
+    private String pemLoc = "src/main/resource/randomCivilianKey.pem";
+    private String passphrase = "passphrase";
+
     public static void main(String... args) {
         new Issue2319Spec().runSuiteSync();
     }
 
     @Override
     public List<HapiSpec> getSpecsInSuite() {
         return List.of(new HapiSpec[] {
-            sysFileSigReqsWaivedForMasterAndTreasury(),
-            sysAccountSigReqsWaivedForMasterAndTreasury(),
-            propsPermissionsSigReqsWaivedForAddressBookAdmin(),
             sysFileImmutabilityWaivedForMasterAndTreasury(),
+            propsPermissionsSigReqsWaivedForAddressBookAdmin(),
+            sysAccountSigReqsWaivedForMasterAndTreasury(),
+            sysFileSigReqsWaivedForMasterAndTreasury()
         });
     }
 
     @HapiTest
-    @Disabled("Failing or intermittently failing HAPI Test")
     final HapiSpec propsPermissionsSigReqsWaivedForAddressBookAdmin() {
-        var pemLoc = "<PEM>";
-
         return defaultHapiSpec("PropsPermissionsSigReqsWaivedForAddressBookAdmin")
                 .given(
-                        keyFromPem(pemLoc).name("persistent").simpleWacl().passphrase("<SECRET>"),
+                        keyFromPem(pemLoc).name("persistent").passphrase(passphrase),
```

Comment:
Curious, why was `.simpleWacl()` removed?

Reply:
I could not get it to properly load the key from the pem file with `.simpleWacl()` and the test failed with `INCORRECT_SIGNATURE`. It's possible that the format of the pem file is wrong, but I could not find documentation on how it should be generated.

Reply:
@tinker-michaelj or @Neeharika-Sompalli this test fails against mono service locally. Not sure if it's run in CI or not. Is it ok to change these tests?

Reply:
Thanks. Addressed.